src/stores/results.ts

```diff
@@ -17,27 +16,49 @@ export const useResultsStore = defineStore("results", {
         logoPath: "/img/benefits/logo-bafa-bafd.png",
         description:
           "Différents organismes peuvent vous aider à financer votre formation BAFA ou BAFD.",
-        redirectionPage: "bafa-bafd",
+        interestFlag: "_interetBafa",
       }
 
+      const veloGroup: BenefitGroup = {
+        benefits: [],
+        id: "velo-group",
+        label: "Aides à l'achat d'un vélo",
+        logoPath: "/img/benefits/logo-velo.jpg",
+        description:
+          "Différents organismes peuvent vous aider à financer votre vélo.",
+        interestFlag: "_interetsAidesVelo",
+      }
+
+      const groups = [bafaGroup, veloGroup]
```

Comment:
```suggestion
      const groups : BenefitGroup[] = [{...bafaGroup}, {...veloGroup}
```

Où `...xx` serait les objets (lignes 12 et 22).